crates/ruma-server-util/src/media.rs

```diff
@@ -0,0 +1,37 @@
+//! Functions for implementing a media repo.
+
+/// Checks whether a given `Content-Type` is considered "safe" for having a `Content-Disposition` of
+/// `inline`. This should be used on `/download` only, as `/thumbnail` should only ever return safe
+/// types as thumbnails are generated by the server.
+pub fn is_safe_content_type(content_type: &str) -> bool {
```

Comment:
I believe this should be in the `ruma-client-api` crate. It can be useful for clients too, if they want to accept only those content types.

Reply:
Also, should I have the function directly inside `media.rs`, or should it be in it's own separate module?

Reply:
It's fine directly in the `media` module. For the docs you can change it to `Endpoints and helpers for the media repository`, but its fine if it's not changed.